### PR TITLE
Add post closure guidance for author

### PR DESCRIPTION
**Pull request by ArtOfCode-**
meta:292486

### CODE REVIEW

**Comment by cellio** [comment]
Looks good to me (based on inspection, not testing).

**Review by Oaphi — APPROVED** [review]
>  Your post has been closed and is not accepting new answers. You should read the reason for closure listed below and check for any comments left by others about why your post has been closed and what it needs, then edit your post to improve it to a state where it can be reopened. If you need help, reply to comments to clarify any requests, or join us in chat for more guidance.

I'd slightly reword it to (+ split in two paragraphs and add a link to chat [presumably, our community server on Discord?]):

>  Your post has been closed and is not accepting new answers. Please read the reason below and check any comments left by others about why your post has been closed and what it needs, then edit your post to improve it to a state where it can be reopened. 
> 
> If you need help, reply to comments to clarify any requests, or join us in chat for more guidance.

Other than possibly doing that, LGTM
